components/Frame/Header.js

```diff
@@ -33,6 +37,7 @@ const SEARCH_BUTTON_WIDTH = 28
 const styles = {
   bar: css({
     zIndex: ZINDEX_HEADER,
+    position: 'fixed',
```

Comment:
Not necessarily related to this line of code, but:
On Chrome/Android, when scrolling down on the page the header is cut off by a few pixels:
![screenshot_20180823-115447](https://user-images.githubusercontent.com/23520051/44518954-f7c11c00-a6cb-11e8-95e7-21102d95a98f.png)

Reply:
can't reproduce